### PR TITLE
Bugfix/#2914 news redesign & #2732 search bug

### PR DESCRIPTION
**Pull request by tkhristina**
- fix bug "Two separate search result links are displayed instead of one 'See all results' after searching"

before:
image

after:
image

- change design of the searching results 

before:
image
image

after:
image
image

- remove "item founds" when results are loading 

before:
image

after:
image

- fix display when the elements are not found

before:
image

after:
image

- create mock data to display preview text

### CODE REVIEW

File: src/app/main/component/layout/components/search-item/search-item.component.html [hunk 1]
```diff
@@ -9,14 +9,20 @@
...
+  <!-- <div class="search_item-data-added-news">
+  <p class="search_item-text">
+    <img class="search_item-icon" [src]="profileIcons.calendarIcon" alt="calendar" />
+    {{ searchModel.creationDate | date }}
+  </p>
+  <p class="user-data-text-date-share">
+    <img class="search_item-icon" [src]="profileIcons.userIcon" alt="user" />
+    {{ searchModel.author.name }}
+  </p>
+</div> -->
```
Comment (dyadyushko) [inline src/app/main/component/layout/components/search-item/search-item.component.html:27]: Is it possible to remove this comments

File: src/app/main/component/layout/components/search-not-found/search-not-found.component.scss [hunk 2]
```diff
@@ -5,14 +5,18 @@ p {
...
 .search_not-found-text,
 .search_not-found-header {
-  width: 350px;
+  // width: 350px;
   font-family: var(--secondary-font);
   font-weight: 500;
   font-size: 1.3rem;
   line-height: 28px;
-  word-break: break-all;
+  // word-break: break-all;
```
Comment (dyadyushko) [inline src/app/main/component/layout/components/search-not-found/search-not-found.component.scss:19]: these ones as well

File: src/app/main/component/layout/components/search-not-found/search-not-found.component.scss [hunk 3]
```diff
@@ -21,16 +25,16 @@ p {
...
   margin-top: 10px;
-
   a:hover {
     text-decoration: none;
   }
 }
 
 .search_not-found-image {
-  border-radius: 5px;
+  // border-radius: 5px;
```
Comment (dyadyushko) [inline src/app/main/component/layout/components/search-not-found/search-not-found.component.scss:37]: same

File: src/app/main/component/layout/components/search-not-found/search-not-found.component.scss [hunk 4]
```diff
@@ -52,10 +58,11 @@ p {
...
   max-width: 360px;
-  height: 500px;
-  background: #fff;
-  box-shadow: 0 0.25rem 2rem #e6ebea;
-  border-radius: 0.25rem;
+  min-height: 352px;
+  background: var(--primary-white);
+  // box-shadow: 0 0.25rem 2rem #e6ebea;
+  box-shadow: 1px 4px 8px rgba(100, 114, 125, 0.18);
+  // border-radius: 0.25rem;
```
Comment (dyadyushko) [inline src/app/main/component/layout/components/search-not-found/search-not-found.component.scss:65]: same

File: src/app/main/service/search/search.service.ts [hunk 5]
```diff
@@ -18,7 +19,8 @@ export class SearchService {
   public allElements: SearchDto;
 
   public getAllResults(searchQuery: string, lang: string): Observable<SearchModel> {
-    return this.http.get<SearchModel>(`${this.backEndLink}search?searchQuery=${searchQuery}&lang=${lang}`);
+    return this.http.get<SearchModel>(backendMockSearchAll);
+    // (`${this.backEndLink}search?searchQuery=${searchQuery}&lang=${lang}`);
```
Comment (dyadyushko) [inline src/app/main/service/search/search.service.ts:23]: same

File: src/app/main/service/search/search.service.ts [hunk 6]
```diff
@@ -28,9 +30,8 @@ export class SearchService {
     sort: string = '',
     items: number = 9
   ): Observable<SearchDataModel> {
-    return this.http.get<SearchDataModel>(
-      `${this.backEndLink}search/${category}?searchQuery=${query}&sort=${sort}&page=${page}&size=${items}`
-    );
+    return this.http.get<SearchDataModel>(backendMockSearchNews);
+    // ()`${this.backEndLink}search/${category}?searchQuery=${query}&sort=${sort}&page=${page}&size=${items}`);
```
Comment (dyadyushko) [inline src/app/main/service/search/search.service.ts:34]: same

Comment (tkhristina) [reply src/app/main/service/search/search.service.ts]: fixed

File: src/app/main/service/search/search.service.ts (hunk 5 above)
Comment (tkhristina) [reply src/app/main/service/search/search.service.ts]: fixed

File: src/app/main/component/layout/components/search-not-found/search-not-found.component.scss (hunk 4 above)
Comment (tkhristina) [reply src/app/main/component/layout/components/search-not-found/search-not-found.component.scss]: fixed

File: src/app/main/component/layout/components/search-not-found/search-not-found.component.scss (hunk 3 above)
Comment (tkhristina) [reply src/app/main/component/layout/components/search-not-found/search-not-found.component.scss]: fixed

File: src/app/main/component/layout/components/search-not-found/search-not-found.component.scss (hunk 2 above)
Comment (tkhristina) [reply src/app/main/component/layout/components/search-not-found/search-not-found.component.scss]: fixed

File: src/app/main/component/layout/components/search-item/search-item.component.html (hunk 1 above)
Comment (tkhristina) [reply src/app/main/component/layout/components/search-item/search-item.component.html]: removed

**Comment by JimHolman1199** [comment]
please fix linter

**Review by dyadyushko — APPROVED** [review]

**Pull request merged**